tools/src/main/java/com/nedap/archie/archetypevalidator/validations/SpecializedOccurrencesValidation.java

```diff
@@ -0,0 +1,140 @@
+        MultiplicityInterval childNodeOccurrences = childCObject.effectiveOccurrences(combinedModels::referenceModelPropMultiplicity);
+
+        if(parentCObject.getNodeId().equals(childCObject.getNodeId()) && parentNodeOccurrences.equals(childNodeOccurrences)) {
+            //this is the parent node appearing in the flattened child archetype without change in occurrence. That is guaranteed to be valid
+            return ConformanceCheckResult.conforms();
+        }
+
+        List<CObject> allRedefinedNodes = childCObject.getParent().getChildren().stream().filter(
+                child -> child.nodeIdConformsTo(parentCObject)
+        ).collect(Collectors.toList()); // All child nodes in the child archetype
+        MultiplicityInterval allRedefinedNodeOccurrencesSummed = new MultiplicityInterval(0, 0);
+        for (CObject childNode : allRedefinedNodes) {
+            if (allRedefinedNodeOccurrencesSummed.isOpen()) {
+                break;
+            }
+            MultiplicityInterval redefinedOccurrences = childNode.effectiveOccurrences(combinedModels::referenceModelPropMultiplicity);
+            if (!allRedefinedNodeOccurrencesSummed.isLowerUnbounded()) {
+                Integer lower = allRedefinedNodeOccurrencesSummed.getLower();
+                if (!redefinedOccurrences.isLowerUnbounded()) {
+                    allRedefinedNodeOccurrencesSummed.setLower(lower + redefinedOccurrences.getLower());
+                } else {
+                    allRedefinedNodeOccurrencesSummed.setLowerUnbounded(true);
+                    allRedefinedNodeOccurrencesSummed.setLower(null);
+                }
+            }
+            if (!allRedefinedNodeOccurrencesSummed.isUpperUnbounded()) {
+                Integer upper = allRedefinedNodeOccurrencesSummed.getUpper();
+                if (!redefinedOccurrences.isUpperUnbounded()) {
+                    allRedefinedNodeOccurrencesSummed.setUpper(upper + redefinedOccurrences.getUpper());
+                } else {
+                    allRedefinedNodeOccurrencesSummed.setUpperUnbounded(true);
+                    allRedefinedNodeOccurrencesSummed.setUpper(null);
+                }
+            }
+        }
+        MultiplicityInterval cardinality = childCObject.getParent().getCardinality() == null ? null : childCObject.getParent().getCardinality().getInterval();
+        if(cardinality == null) {
+            cardinality = combinedModels.referenceModelPropMultiplicity(childCObject.getParent().getParent().getRmTypeName(), childCObject.getParent().getRmAttributeName());
+        }
+        if(cardinality != null && !cardinality.isUpperUnbounded() &&
+                (allRedefinedNodeOccurrencesSummed.isUpperUnbounded() || allRedefinedNodeOccurrencesSummed.getUpper() > cardinality.getUpper())) {
+            allRedefinedNodeOccurrencesSummed.setUpperUnbounded(false);
+            allRedefinedNodeOccurrencesSummed.setUpper(cardinality.getUpper());
+        }
```

Comment:
It is necessary.

If you do in the parent:

```
ELEMENT[id5] matches {
  value matches {
     DV_TEXT[id6]
  }
}
```

And in a child you want to do:

```
ELEMENT[id5] matches {
  value matches {
     DV_TEXT[id6.1]
     DV_CODED_TEXT[id6.1] matches {....}
  }
}
```

It will fail without this check. Of course it is possible to do this without a specialization in this case, but it would break existing archetypes, plus in this way you can now also create a cluster with `items cardinality matches {1}`, with several alternatives.

Reply:
Added a test for this